boundary_test.go

```diff
@@ -477,6 +477,24 @@ func TestBoundaryReaderReadErrors(t *testing.T) {
 	}
 }
 
+func TestBoundaryReaderLongLine(t *testing.T) {
+	//dest := make([]byte, 8 *1024)
```

Comment:
remove

Reply:
Oops, removed. I can squash the fixes and force push as a single commit if you want. I'm not sure if github can do this automatically like gitlab.